script/vsts/platforms/linux.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   steps:
   - script: script/build --create-debian-package --create-rpm-package --compress-artifacts
     env:
+      CI: true
```

Comment:
I could have sworn I read somewhere in Azure Pipelines docs that this is already set, but now I can't find it :/

Reply:
Haha also adding this broke everything 🙃